enterprise/api/v1/urls.py

```diff
@@ -57,6 +57,11 @@
         views.EnterpriseSubsidyFulfillmentViewSet.as_view({'post': 'cancel_enrollment'}),
         name='enterprise-subsidy-fulfillment-cancel-enrollment'
     ),
+    re_path(
+        r'enterprise-subsidy-fulfillment/recently_unenrolled/?$',
```

Comment:
@iloveagent57 Thoughts on the restful-nes of this? I think that one of our goals is to remain restful in our api implementation? Should we be using an endpoint like `enterprise-course-enrollment` and sending in params to filter on `learner credit enrollments` and `order_by` pick the records we want? 

As I'm reading it this seems a bit overly specific and like an RPC?

Reply:
The way I would think about what Brian is proposing:
* Implement this as an `EnterpriseSubsidyFulfillment` list view.
* Allow that view to be filtered on the `unenrolled` boolean and filtered by the `modified` datetime.

This will give clients of the endpoint more flexibility in how they request data (i.e. let the client decide what "recent" means).

This brings up another point - perhaps `unenrolled` should really be a datetime field called `unenrolled_at`?